meetings/2022-02-24.md

```diff
@@ -0,0 +1,59 @@
+# W3C Solid Community Group: Notifications Panel
+
+* Date: 2022-02-24T15:00:00Z
+* Call: https://meet.jit.si/solid-notifications
+* Chat: https://gitter.im/solid/notifications-panel
+* Repository: https://github.com/solid/notifications-panel
+* Status: Published
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+* TimBL
+* Rahul Gupta
+* Aaron Coburn
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+---
+
+## Topics
```

Comment:
Would it make sense to be more explicit that no notes were taken and that this reflects the agenda but not the discussion?

Reply:
That's perfect, thanks.